content/english/linux-basics/activity-2.md

```diff
@@ -7,9 +7,32 @@ difficulty: "Intermediate"
 weight: 5
 ---
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/v1XLyLuQQyA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
 *So before we can stop the hackers, we need to learn to navigate the Linux filesystem.*
 
-### Where in the file system are you?
+### What is a filesystem? 
+
+A filesystem is a structured collection of files and data on a computer disk or drive. The highest level of a filesystem is referred to as the "root". You can think of it as the beginning of the filesystem. 
+
+{{% notice tip %}}
+Fun fact: filesystems are often described as a "tree". This is because the filesystem looks like an upside down tree with each folder as a branch or leaf. For example, your filesystem may look like the below.
+
+![picture of a filesystem tree](../images/filesystem-tree.png?classes=border,shadow)
+{{% /notice %}}
+
+For example, on a windows computer, you can browse the computer's "C:\" drive filesystem in the "file explorer" by navigating to the below
```

Comment:
```suggestion
On a Windows computer, you can browse the computer's "C:\" drive filesystem in the "file explorer" like below
```

Reply:
Do we need to talk about what the C:\ drive is?

Reply:
As in talking about what a drive is or as in talking about the significance of the C:\ drive itself? If the former maybe it's best to take out the word "driver". Otherwise, if we want to explain the significance, then maybe we could add that as like a fun fact or something?